src/pluginHooks.js

```diff
@@ -669,6 +669,8 @@ const REGISTERED_HOOKS = [
    * @param {CellRange} sourceRange The range values will be filled from.
    * @param {CellRange} targetRange The range new values will be filled into.
    * @param {string} direction Declares the direction of the autofill. Possible values: `up`, `down`, `left`, `right`.
+   * @param {boolean} hasFillDataChanged Indicates whether the original fillData has been changed in
+   *                                     the `beforeAutofill` hook.
```

Comment:
If the last argument is going to be removed, why would we add it to the API docs? Shouldn't it be private for internal use?

Reply:
In the first iteration of the PR, I didn't want to mention it. But, after a while, I thought ... why not it can be useful for developers.

Reply:
If you want to remove the argument in the future I'd remove that line. I'm ok with that.

Reply:
I don't, you do 😄 

https://github.com/handsontable/handsontable/blob/7981bbbe48234f4ae19109b1fa76cb1f335bad32/src/plugins/autofill/autofill.js#L257-L261

Reply:
Done within d10fcb3cb0327b2498ebb8a79bf17ede52ea8c8e